packages/markdown-pdf/src/PdfTransformer.test.js

```diff
@@ -86,8 +129,11 @@ describe('pdf import', () => {
     getPdfFiles().forEach(([file, pdfContent], i) => {
```

Comment:
This whole thing needs an `await`, or the subsequent `getMarkdownFiles()` needs to be in a `then`.

Reply:
Not sure I understand. Those are separate tests. Jest will await tests that return promises automatically.

Reply:
Doesn't the markdown test need to wait for the files to be written? (I may have misunderstood the structure of the tests).

Reply:
No, the PDFs the tests write out are just to help debugging. There are separate PDFs that are committed that are the input to the subsequent tests.